mappers/Baseline.rb

```diff
@@ -245,7 +245,7 @@ def time_mapping(time)
 
 
             # ChangeBuildingLocation
-            # set skip measure to false change building location
+            # set skip to false for change building location
             OpenStudio::Extension.set_measure_argument(osw, 'ChangeBuildingLocation', '__SKIP__', false)
```

Comment:
@kflemin if it doesn't find the weather file from the Feature File and does't find the epw in the weather it will look for the default argument in the in the base_workflow.osw measure.